data.sql

```diff
@@ -8,4 +8,25 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Pikachu', '2021-01-07', 1, false, 15.04);
 
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
-VALUES ('Devimon', '2017-05-12', 5, true, 11);
\ No newline at end of file
+VALUES ('Devimon', '2017-05-12', 5, true, 11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES ('Charmander', '2020-02-08', 0, false, -11, 'Fire');
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES ('Plantmon', '2021-11-15', 2, true, -5.7, 'Grass');
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES ('Squirtle', '1993-04-02', 3, false, -12.13, 'Water');
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES ('Angemon', '2005-06-12', 1, true, -45, 'Angel');
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES ('Boarmon', '2005-06-07', 7, true, 20.4, 'Mammal');
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES ('Blossom', '1998-10-13', 3, true, 17, 'Flower');
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES ('Ditto', '2022-05-14', 4, true, 22, 'Normal');
```

Comment:
- [ ] Please you are inserting values into the species column which wasn't required in the project requirement so I think you should not insert any values to the species column when inserting these data (The species column should be left null) :pray:

Reply:
> * [x]  Please you are inserting values into the species column which wasn't required in the project requirement so I think you should not insert any values to the species column when inserting these data (The species column should be left null) pray

- [REQUIRED] Kindly attend to this required change from the previous reviewer. It is yet to be attended to. Thanks